.github/CONTRIBUTING.md

```diff
@@ -91,24 +105,36 @@ The stylesheet available for use within DM can be found in `code/stylesheet.dm`.
 In order to make `Exited()` and `Entered()` procs more reliable, the usage of `forceMove()` when forcibly moving one item to another location, be it another item or turf, is required. Directly changing an item's loc values will skip over calls to the aforementioned procs, thus making them less useful and more unreliable.
 
 An example of improper item moving:
-```
+```DM
 /proc/some_proc(var/obj/A, var/obj/B)
 	A.loc = B	// Simply move A inside B.
 ```
 
 An example of proper item moving:
-```
+```DM
 /proc/some_proc(var/obj/A, var/obj/B)
 	A.forceMove(B)	// This will call A.loc.Exited() and B.Entered().
 					// The first method does not call either of those.
 ```
 
+### Regarding the variable usr
+If at all possible, procs outside of verbs and `Topic()` should avoid reliance on `usr`, and instead use a custom argument to specify the user and its expected type. This makes it easier to reuse procs in chains where `usr` is not always defined, and combats against potential security flaws that relying on `usr` can bring.
```

Comment:
Some BYOND procs like `Click()` have valid `usr` too.

Reply:
But then I'd have to specify that Click and our own Click wrappers/end-points are different, and so on. Basically, the admin in me is saying to let those edgecases be, the people who go into that code will know enough to make the distinction.

Reply:
Technically `usr` should be valid in those too if they originate from BYOND `Click()` callstacks.